WeScan/ImageScannerController.swift

```diff
@@ -62,10 +73,30 @@ public final class ImageScannerController: UINavigationController {
         fatalError("init(coder:) has not been implemented")
     }
     
+    private func setupConstraints() {
+        let blackFlashViewConstraints = [
+            blackFlashView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
```

Comment:
Nit: Having the same order as UIEdgeInsets arguments would be nicer: Top, Left, Bottom, Right.